src/content/api/_endpoints/managed-integrations-list-by-accountId.js

```diff
@@ -0,0 +1,49 @@
+export default {
+  method: 'GET',
+  path: '/api/accounts/Jaim1Cu1Q55uooxSens6yk/managed-integrations',
+  request: {
+    headers: {
+      'X-Api-Key': '<TOKEN>',
+    },
+  },
+  response: [
```

Comment:
@pupato13 one last thing - I think our query and response should include pagination so that readers of our doc know this endpoint handles pagination.

See src/content/api/_endpoints/collections-list.js for an example